FILESTRUCTURE.md

```diff
@@ -0,0 +1,67 @@
+# Vue File Structure
```

Comment:
Could simply say at the start that we follow Vue's default file structure.

Reply:
changing this to Nuxt, and since Nuxt file structure is a bit odd and not the norm for many frameworks like base Vue, React, Angular, etc I think it's fine to have it and keep it as it's own MD that if someone wants to read they can